kerchunk/combine.py

```diff
@@ -404,13 +388,32 @@ def translate(self, filename=None, storage_options=None):
             self.store_coords()
         if 3 not in self.done:
             self.second_pass()
-        out = self.consolidate()
+        if 4 not in self.done:
+            if self.postprocess is not None:
+                self.out = self.postprocess(self.out)
```

Comment:
Please alter the docstring for postprocess, to add its signature `postprocess(dict)->dict`.